src/System.Private.CoreLib/src/System/Runtime/Bootstrap/BootstrapHelpers.cs

```diff
@@ -0,0 +1,61 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+using System.Security;
+using System.Text;
+using Internal.NativeFormat;
+using System.Runtime.Bootstrap;
+
+namespace System.Runtime
```

Comment:
We have been placing compiler support classes into the Internal.Runtime.CompilerHelpers namespace, mostly so that we don't pollute the public namespace with internal things too much. I'm not sure how much objective value there is in doing it, but feels like a right thing to me...